src/framework/tours/internal/toursservice.cpp

```diff
@@ -88,11 +100,26 @@ void ToursService::initTours()
             }
 
             TourStep step;
-            step.title = itemObj.value("title").toString();
-            step.description = itemObj.value("description").toString();
             step.videoExplanationUrl = itemObj.value("video_explanation_url").toString();
             step.controlUri = Uri(itemObj.value("control_uri").toString());
 
+            JsonObject itemLocaleObj = itemObj.value("locale").toObject();
+
+            if (!itemLocaleObj.contains(locale)) {
```

Comment:
Do we still need this code?

Reply:
I will keep this type of registration for the future